src/issue.ts

```diff
@@ -52,92 +52,101 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
   const result = {issues: new Array<Issue>(), repo};
   let res!: GaxiosResponse<IssuesApiResponse>;
   const rootUrl = 'https://drghs.endpoints.devrel-prod.cloud.goog/api/v1';
-  const url = `${rootUrl}/${repo.repo}/issues?key=${apiKey}&closed=false`;
-  try {
-    res = await request<IssuesApiResponse>({url});
-  } catch (e) {
-    console.warn(`Error fetching issues for ${repo.repo}.`);
-    //console.warn(e);
-    return result;
-  }
 
-  if (!res.data || !res.data.issues) {
-    return result;
-  }
+  let pageToken = '';
+  while (pageToken === '' && result.issues.length < 1) {
+    const url = `${rootUrl}/${repo.repo}/issues?key=${apiKey}&closed=false${
+      pageToken === '' ? '' : 'page=' + pageToken
+    }`;
+
+    try {
+      res = await request<IssuesApiResponse>({url});
+    } catch (e) {
+      console.warn(`Error fetching issues for ${repo.repo}.`);
+      // console.warn(e);
+      return result;
+    }
+
+    if (!res.data || !res.data.issues) {
+      return result;
+    }
 
-  res!.data.issues.forEach(r => {
-    const api = getApi(r, repo);
-    const issue: Issue = {
-      owner,
-      name,
-      language: repo.language,
-      repo: repo.repo,
-      types: getTypes(r),
-      api,
-      team: getTeam(r.repo, api),
-      isOutOfSLO: isOutOfSLO(r),
-      isTriaged: isTriaged(r),
-      pri: r.priorityUnknown ? undefined : getPriority(r.priority),
-      isPR: r.isPr,
-      number: r.issueId,
-      createdAt: r.createdAt,
-      title: r.title,
-      url: r.url,
-      labels: r.labels || [],
-      assignees: r.assignees ? r.assignees.map(x => x.login) : [],
-    };
-
-    let use = true;
-    if (flags) {
-      if (flags.api) {
-        const apiTypes = flags.api
-          .split(',')
-          .map(t => t.trim())
-          .filter(t => t.length > 0);
-        if (!issue.api || apiTypes.indexOf(issue.api) === -1) {
+    pageToken = res!.data.nextPageToken;
+
+    res!.data.issues.forEach(r => {
```

Comment:
I find the mixing of callbacks and `async`/`await` a little hard to read, what if we go with:

```
const issues = res!.data.issues;
for (const issue of issues) {
 ...
}
```